server.js

```diff
@@ -27,22 +22,26 @@ app.get('/songs', function(req, res) {
   res.render('songs');
 });
 
-app.get('/artists:artist_id', function(req, res) {
-  res.render('artist');
-});
 
-app.get('/albums:album_id', function(req, res) {
-  res.render('album');
+//Used to grab the static files
+app.use(express.static(path.join(__dirname + 'public')));
+
+var albums = app.locals.albums = JSON.stringify(require("./public/albums.json"));
```

Comment:
Oh strange! I've never seen it done this way. Okay, I would suggest to you that you require it up top and pass the variable name to ```JSON.stringify()```